config/routes.rb

```diff
@@ -1,8 +1,4 @@
 Rails.application.routes.draw do
-  get 'admin/index'
```

Comment:
what is this change and why is it in this PR?

Reply:
@ritika1206 please add reply in all the comments stating what you've done to resolve that particular comments